tesseract_ros/tesseract_msgs/msg/ContactResult.msg

```diff
@@ -1,11 +1,13 @@
 time stamp
 float64 distance
-byte[2] type_id # ROBOT_LINK = 0, ROBOT_ATTACHED = 1
+byte[2] type_id
 string[2] link_names
 uint64[2] shape_id # Link's child shape index
 uint64[2] subshape_id # If mesh or octomap this will be the voxel/triangle index
+geometry_msgs/Pose[2] transform # Transform of link_names at time step 0
```

Comment:
I realize this is super picky, but these are in a different order from the class which makes it a bit difficult to compare. Since this doesn't have documentation it would be nice if it was the same order as in types.h